e2e/autoscript-suites/src/utilities/setup-and-go.ts

```diff
@@ -69,12 +69,8 @@ export async function setupAndGo(
   await page.goto(url.toString());
 
   // Listen for events on page
-  page.on('console', (msg) => {
-    messageArray.push(msg.text());
-  });
-
-  page.on('console', (msg) => {
-    messageArray.push(msg.text());
+  page.on('console', async (msg) => {
```

Comment:
docs https://playwright.dev/docs/api/class-page#page-event-console show this callback is async, and we had two identical blocks of code, so i cleaned it up